src/Backend/Core/Engine/TwigTemplate.php

```diff
@@ -2,10 +2,18 @@
 
 namespace Backend\Core\Engine;
 
-use Backend\Core\Engine\Language as BL;
+use Backend\Core\Language\Language as BL;
```

Comment:
To be consistent, can you use "BackendLanguage" instead of "BL" and in all the translations in this file?